drivers/storage/portworx/util/util.go

```diff
@@ -179,6 +181,9 @@ const (
 	// EnvKeyPortworxServiceAccount key for the env var which tells custom Portworx
 	// service account
 	EnvKeyPortworxServiceAccount = "PX_SERVICE_ACCOUNT"
+	// EnvKeyPortworxServiceAccountTokenRefreshIntervalHours key for the env var which tells how often
+	// the token associated with the Px service account gets refreshed
+	EnvKeyPortworxServiceAccountTokenRefreshIntervalHours = "PX_SERVICE_ACCOUNT_TOKEN_REFRESH_INTERVAL_HOURS"
```

Comment:
Putting "hours" here limits the interval to be at least 1 hour. But it's also probably not a good idea for user to put 30 seconds here I assume? What would be a lower bound of this interval?

Reply:
Can we use "Minutes"? I was hoping to use this setting from a functional test to configure lower during for testing.

Reply:
Oh yeah right